styles.css

```diff
@@ -494,3 +499,403 @@ header nav a {
+    margin: 0;
+  }
+
+  .get-intouch input {
+    width: 80%;
+  }
+
+  .get-intouch textarea {
+    width: 80%;
+  }
+}
+
+@media (min-width: 1024px) {
+  main {
+    background-image: url("./assets/images/header_desktop.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    height: 95vh;
+    margin-top: -62px;
+  }
+
+  .flex p {
+    width: 60%;
+  }
+
+  .flex h1 {
+    font-size: 40px;
+    margin: 50px 0 20px 0;
+    padding-top: 0;
+  }
+
+  header {
+    position: relative;
+  }
+
+  header a {
+    display: none;
+  }
+
+  header nav ul {
+    position: absolute;
+    top: 0;
+    left: 30%;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-column-gap: 15px;
+    margin: 0 20px;
+  }
+
+  .about-me {
+    background-image: url('./assets/images/about_me2.png');
+    background-size: 254px 138px;
+    background-repeat: no-repeat;
+    height: 347px;
+    margin-top: 15%;
+    position: relative;
+  }
+
+  .upper-bg {
+    background-image: url('./assets/images/about_me1.png');
+    background-size: 167px 210px;
+    background-repeat: no-repeat;
+    right: -20px;
+    width: 188px;
+    height: 207px;
+    margin-top: -104px;
+    position: absolute;
+  }
+
+  .about-me h4 {
+    font-size: 2.7rem;
+    position: absolute;
+    top: -100px;
+    left: 15%;
+  }
+
+  .about-me button {
+    width: 18%;
+    margin-left: 42%;
+    margin-top: 40px;
+  }
+
+  .about-me p {
+    font-size: 1.3rem;
+    width: 60%;
+    margin: 40px 24% 10px;
+  }
+
+  .grid-ul {
+    margin: 0 20px;
+  }
+
+  .card-section {
+    grid-column: 1/4;
+  }
+
+  .card-section1 {
+    grid-column: 4/7;
+  }
+
+  .card-section2 {
+    grid-column: 7/10;
+    margin-left: 0;
+  }
+
+  .get-intouch {
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    margin: 8% 40px 20% 40px;
+  }
+
+  .get-intouch h4 {
+    width: 60%;
+    font-size: 1.7rem;
+  }
+
+  .get-intouch input {
+    width: 80%;
+  }
+
+  .get-intouch textarea {
+    width: 80%;
+  }
+
+  .get-intouch ul {
+    position: absolute;
+    bottom: -300px;
+    left: 35%;
+    padding: 20px;
+  }
+
+  hr {
+    border: 1px solid #dfe1e6;
+  }
+
+  .get-intouch span {
+    display: none;
+  }
+
+  .form-img {
+    background-image: url('./assets/images/contact_form.png');
+    background-repeat: no-repeat;
+    width: 117px;
+    height: 175px;
+    position: absolute;
+    right: 793px;
+    bottom: -188px;
+  }
+
+  .left-bg {
+    background-image: url('./assets/images/illustrate_bg.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+    top: 226px;
+    left: -37px;
+    width: 100%;
+    height: 340px;
+    position: absolute;
+  }
+
+  .right-bg {
+    background-image: url('./assets/images/form_bg.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+    position: absolute;
+    top: 99px;
+    left: 767px;
+    width: 214px;
+    height: 465px;
+  }
```

Comment:
- [ ] Kindly use percentage `%` because your background positions are not right when you try with a bigger width.

![image](https://user-images.githubusercontent.com/35535130/155413878-4bb6a168-3c3b-4ac8-926c-4bd568d03ba4.png)